pkg/task/task.go

```diff
@@ -1053,6 +1056,28 @@ func (m *taskExecutor) deleteOEV1alpha1CSV() (err error) {
 	return
 }
 
+// execCoreV1Pod runs given command remotely in given container of given pod
+// and post stdout and and stderr in JsonResult. You can get it using -
+// {{- jsonpath .JsonResult "{.Stdout}" | trim | saveAs "XXX" .TaskResult | noop -}}
+func (m *taskExecutor) execCoreV1Pod() (err error) {
+	podexecoptsFromTemplate, err := m_k8s_podexec.WithTemplate("execCoreV1Pod", m.runtask.Spec.Task, m.templateValues)
```

Comment:
the names used here can be single character names. e.g.:
```go
e, err := podexec.WithTemplate("taskname", taskYaml, values)
if err != nil {
  return err
}
podExec, err := e.AsAPIPodExec()
if err != nil {
  return err
}
```
Above does not hinder the readability IMO

Alternatively, if we handle error as a value this can become:
```go
execOpts, err := podexec.
       WithTemplate("taskname", taskYaml, values).
       AsAPIPodExec()
if err != nil {
  return err
}
```

Reply:
done